### PR TITLE
Update product-image-normalizer.ts to allow external product images

### LINKED ISSUE #5559: Issue with complete media URL
if from Backend we receive full product image URL then in `ProductImageNormalizer` it is concatenated with baseURL. Which is wrong.
for example if image have following url 

`https://externalImageServer/images/product/001.jpg`
and base Url ist `localhost:9002` then the final image url is produces as 

`https://localhost:9002https://externalImageServer/images/product/001.jpg`

Problem is here in class --> ProductImageNormalizer at linke 57
```
image.url =
          (this.config.backend.media.baseUrl ||
            this.config.backend.occ.baseUrl ||
            '') + image.url;
```
## Environment Details

- Spartacus:  1.3.*

## Offer to Fix
YES

### PR DESCRIPTION
**Pull request by kbalagtey-tacit**
As per the media service, it would be good to allow externally hosted product images

fixes #5559

### CODE REVIEW

**Comment by Xymmer** [comment]
i've asked our architect so we're sure there's no reason not to do this

File: projects/core/src/occ/adapters/product/converters/product-image-normalizer.ts [hunk 1]
```diff
@@ -54,10 +54,13 @@ export class ProductImageNormalizer implements Converter<Occ.Product, Product> {
          * `backend.media.baseUrl` by default, but fallback to `backend.occ.baseUrl`
          * if none provided.
          */
-        image.url =
-          (this.config.backend.media.baseUrl ||
-            this.config.backend.occ.baseUrl ||
-            '') + image.url;
+        image.url = image.url.startsWith('http')
```
Comment (tobi-or-not-tobi) [inline projects/core/src/occ/adapters/product/converters/product-image-normalizer.ts]: @kbalagtey-tacit sorry for the slow review here. I like this idea very much. 
Would you be comfortable to add a unit test for this feature? 
You could also move this in a separate method for readability and avoiding such a long method (not your bad).

Comment (tobi-or-not-tobi) [reply projects/core/src/occ/adapters/product/converters/product-image-normalizer.ts]: @kbalagtey-tacit let us know if you're willing to continue here

**Comment by mrSingh007** [comment]
still waiting for this fix.

**Comment by Pucek9** [comment]
Done by #5962

**Pull request closed without merging**